src/components/custom/other-platforms.tsx

```diff
@@ -38,6 +37,41 @@ import Story from 'images/svg/menu/story.svg'
 import SyntheticIndices from 'images/svg/custom/synthetic-indices-nav.svg'
 import TraderTool from 'images/svg/custom/trader-tool-nav.svg'
 
+type CardProps = {
+    is_selected?: boolean
+    word_break_cover?: boolean
+}
+interface DMT5CardProps extends CardProps {
```

Comment:
Since our convention is using `type` instead of `interface`, let's use `type` here like:
```
type DMT5CardProps = CardProps & {
is_ppc_redirect?: boolean
}
```